### PR TITLE
EZEE-1901: Fastly shielding

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/EZEE-1901
| Versions      | all

Also adds missing "Update to latest 4.1" page.

### CODE REVIEW

**Comment by lserwatka** [comment]
@DominikaK @webhdx we must change prefix for this VCLs to `ibexa` for version 4.x

File: docs/updating/from_3.3/update_from_3.3.md [hunk 1]
```diff
@@ -307,3 +307,18 @@ Adapt your `composer.json` file according to [`manifest.json`](https://github.co
 "bazinga:js-translation:dump %PUBLIC_DIR%/assets --merge-domains": "symfony-cmd",
 "ibexa:encore:compile": "symfony-cmd"
 ```
+
```
Comment (MagdalenaZuba) [inline docs/updating/from_3.3/update_from_3.3.md:310]:
```suggestion
```

File: docs/updating/from_3.3/update_from_3.3.md [hunk 2]
```diff
@@ -307,3 +307,18 @@ Adapt your `composer.json` file according to [`manifest.json`](https://github.co
...
+#### VCL configuration for Fastly
+
+Ibexa DXP now supports Fastly shielding. If you are using Fastly and want to use shielding, you need to update your VCL files.
+
+!!! tip
+
+    Even if you do not plan to use Fastly shielding, it is recommended to update the VCL files for future compatibility.
+
+1. Locate the `vendor/ezsystems/ezplatform-http-cache-fastly/fastly/ez_main.vcl` file and update your VCL file with the recent changes.
+2. Do the same with `vendor/ezsystems/ezplatform-http-cache-fastly/fastly/ez_user_cache.vcl`
```
Comment (MagdalenaZuba) [inline docs/updating/from_3.3/update_from_3.3.md:323]:
```suggestion
2. Do the same with `vendor/ezsystems/ezplatform-http-cache-fastly/fastly/ez_user_cache.vcl`.
```

File: docs/updating/from_4.1/update_from_4.1.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
+description: Update your installation to the latest v4.1 version from an earlier v4.1 version.
```
Comment (MagdalenaZuba) [inline docs/updating/from_4.1/update_from_4.1.md:2]:
```suggestion
description: Update your installation to the v4.1.latest version from an earlier v4.1.x version.
```

File: docs/updating/from_4.1/update_from_4.1.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+#### VCL configuration for Fastly
+
+Ibexa DXP now supports Fastly shielding. If you are using Fastly and want to use shielding, you need to update your VCL files.
+
+!!! tip
+
+    Even if you do not plan to use Fastly shielding, it is recommended to update the VCL files for future compatibility.
+
+1. Locate the `vendor/ibexa/fastly/fastly/ez_main.vcl` file and update your VCL file with the recent changes.
+2. Do the same with `vendor/ibexa/fastly/fastly/ez_user_cache.vcl`
```
Comment (MagdalenaZuba) [inline docs/updating/from_4.1/update_from_4.1.md:49]:
```suggestion
2. Do the same with `vendor/ibexa/fastly/fastly/ez_user_cache.vcl`.
```

**Review by MagdalenaZuba — APPROVED** [review]

File: docs/updating/from_2.5/to_3.3.latest.md [hunk 5]
```diff
@@ -156,6 +156,20 @@ Run the following scripts:
...
+#### VCL configuration for Fastly
+
+Ibexa DXP now supports Fastly shielding. If you are using Fastly and want to use shielding, you need to update your VCL files.
+
+!!! tip
+
+    Even if you do not plan to use Fastly shielding, it is recommended to update the VCL files for future compatibility.
+
+1. Locate the `vendor/ezsystems/ezplatform-http-cache-fastly/fastly/ez_main.vcl` file and update your VCL file with the recent changes.
+2. Do the same with `vendor/ezsystems/ezplatform-http-cache-fastly/fastly/ez_user_cache.vcl`
```
Comment (mnocon) [inline docs/updating/from_2.5/to_3.3.latest.md]:
```suggestion
2. Do the same with `vendor/ezsystems/ezplatform-http-cache-fastly/fastly/ez_user_hash.vcl`
```

File: docs/updating/from_2.5/to_3.3.latest.md [hunk 6]
```diff
@@ -156,6 +156,20 @@ Run the following scripts:
...
+#### VCL configuration for Fastly
+
+Ibexa DXP now supports Fastly shielding. If you are using Fastly and want to use shielding, you need to update your VCL files.
+
+!!! tip
+
+    Even if you do not plan to use Fastly shielding, it is recommended to update the VCL files for future compatibility.
+
+1. Locate the `vendor/ezsystems/ezplatform-http-cache-fastly/fastly/ez_main.vcl` file and update your VCL file with the recent changes.
+2. Do the same with `vendor/ezsystems/ezplatform-http-cache-fastly/fastly/ez_user_cache.vcl`.
```
Comment (mnocon) [inline docs/updating/from_2.5/to_3.3.latest.md:170]:
```suggestion
2. Do the same with `vendor/ezsystems/ezplatform-http-cache-fastly/fastly/ez_user_hash.vcl`.
```

**Review by mnocon — CHANGES_REQUESTED** [review]
We need to fix the filename: from `user_cache` to `user_hash`

File: docs/updating/from_3.3/update_from_3.3.md [hunk 7]
```diff
@@ -307,3 +307,17 @@ Adapt your `composer.json` file according to [`manifest.json`](https://github.co
...
+#### VCL configuration for Fastly
+
+Ibexa DXP now supports Fastly shielding. If you are using Fastly and want to use shielding, you need to update your VCL files.
+
+!!! tip
+
+    Even if you do not plan to use Fastly shielding, it is recommended to update the VCL files for future compatibility.
+
+1. Locate the `vendor/ezsystems/ezplatform-http-cache-fastly/fastly/ez_main.vcl` file and update your VCL file with the recent changes.
+2. Do the same with `vendor/ezsystems/ezplatform-http-cache-fastly/fastly/ez_user_cache.vcl`.
```
Comment (mnocon) [inline docs/updating/from_3.3/update_from_3.3.md:322]:
```suggestion
2. Do the same with `vendor/ezsystems/ezplatform-http-cache-fastly/fastly/ez_user_hash.vcl`.
```

File: docs/updating/from_4.1/update_from_4.1.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+#### VCL configuration for Fastly
+
+Ibexa DXP now supports Fastly shielding. If you are using Fastly and want to use shielding, you need to update your VCL files.
+
+!!! tip
+
+    Even if you do not plan to use Fastly shielding, it is recommended to update the VCL files for future compatibility.
+
+1. Locate the `vendor/ibexa/fastly/fastly/ez_main.vcl` file and update your VCL file with the recent changes.
+2. Do the same with `vendor/ibexa/fastly/fastly/ez_user_cache.vcl`.
```
Comment (mnocon) [inline docs/updating/from_4.1/update_from_4.1.md:49]:
```suggestion
2. Do the same with `vendor/ibexa/fastly/fastly/ez_user_hash.vcl`.
```

**Review by mnocon — APPROVED** [review]

**Pull request merged**